src/Octoshift/Services/GithubApi.cs

```diff
@@ -297,7 +297,8 @@ public virtual async Task<string> CreateGhecMigrationSource(string orgId)
         }
         catch (OctoshiftCliException ex) when (ex.Message.Contains("not have the correct permissions to execute"))
         {
-            throw new OctoshiftCliException(ex.Message + INSUFFICIENT_PERMISSIONS_HELP_MESSAGE, ex);
+            var errorSuffix = GenerateInsufficientPermissionsHelpMessage(orgName);
+            throw new OctoshiftCliException(ex.Message + errorSuffix, ex);
```

Comment:
Instead of concatenating two strings it's more efficient to use string interpolation: `$"{ex.Message}. {errorSuffix}"`

Reply:
Thanks! Happy to change that.